src/main/java/net/dv8tion/jda/core/requests/restaction/ChannelAction.java

```diff
@@ -190,6 +186,10 @@ public ChannelAction addPermissionOverride(Role role, Collection<Permission> all
     {
         checkPermissions(allow);
         checkPermissions(deny);
+        if (allow != null)
+            Checks.noneNull(allow, "Granted Permissions");
+        if (deny != null)
+            Checks.noneNull(deny, "Denied Permissions");
```

Comment:
Both of these `noneNull` checks are already being handled by `checkPermissions`, no?